pyproject.toml

```diff
@@ -43,7 +43,6 @@ ws = ["websockets>=15.0.1"]
 mcp = "mcp.cli:app [cli]"
 
 [tool.uv]
-resolution = "lowest-direct"
```

Comment:
Now, we actually want the resolution to be the "highest" - which is the default. Because we have a test job in the pipeline that sets this resolution.

This way we can have the highest resolution on the lockfile, and actually test both highest and lowest-direct.